content/03.results.md

```diff
@@ -138,18 +138,15 @@ Other canonical cancer driver genes were frequently mutated: _BRAF_, _H3F3A_, _C
 Although LGG and embryonal tumors make up the majority of tumor types within the OpenPBTA, most of the significant gene interactions stem from HGATs (N = 847/872). 
 At the broad histology level, _CTNNB1_ significantly co-occurs with _TP53_ (5/14 or 35.7%, OR = 42.9%, q = 1.63e-3) and _DDX3X_ (5/16 or 31.3%, OR = 21.1, q = 4.46e-3) in embryonal tumors, _FGFR1_ and _PIK3CA_ significantly co-occur in LGGs (3/9 or 33.3%, OR = 76.1, q = 3.26e-3), consistent with previous findings [@doi:10.1200/JCO.2010.31.1670; 10.1186/s40478-020-01027-z].
 _TP53_ and _PPM1D_ mutations have been shown to be mutually exclusive in HGATs, and our data recapitulates that trend (52/54 or 96.3% of tumors have a mutation in either gene, OR = 0.188, p = 4.13e-2, q = 5.87e-2) [@doi:10.1038/ng.2938].
+We next assessed the contributions of a brain-specific mutational signature [@doi:10.1038/s43018-020-0027-5] comprising of eight reference signatures (RefSig) across cancer groups **Figure {@fig:Fig3}C** and samples **Figure {@fig:FigS4}A**.
+Stage 0 and/or 1 tumors characterized by low TMBs **Figure {@fig:FigS2}G** such as LGGs, gangliogliomas, craniopharyngiomas, DNETs, and schwannomas are expectedly dominated by Signature 1, which results from the normal process of spontaneous deamination of 5-methylcytosine.
+Signature N6 is CNS-specific signature which we observe nearly universally across samples.
+Drivers of Signature 18, _TP53_, _APC_, _NOTCH1_, are also canonical drivers of medulloblastoma tumors, and indeed, we observe Signature 18 as the most common signature in medulloblastoma tumors.
```

Comment:
I think we should cite this, but it's tricky because this isn't in the paper itself, just on the website database here: https://signal.mutationalsignatures.com/explore/referenceCancerSignature/31/drivers . Maybe just the link in parentheses..?

Reply:
Yeah, I noticed the paper was slightly different and wondered if they updated their website since. I will add.

Reply:
done